python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -120,6 +120,10 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
         check=_validateGalsimInterpolant,
         default="Lanczos(11)",
     )
+    zerothOrderInterpNotEnoughStars = pexConfig.Field[bool](
+        doc="If True, use zeroth order interpolation if not enough stars are found.",
+        default=False
```

Comment:
I suppose `False` is more conservative, but why can't we make this the default?

Reply:
For the moment I don't want to interfere with other processing. I would like to make a proper PR where I add a flag when this is happening and maybe having a set of flag handling other situations in quality of PSF fit. When this is done, I guess indeed it would be ok to switch this to default (my feeling from discussing this in sciunit-psf yesterday).